tests/test_openspiel.py

```diff
@@ -128,16 +131,15 @@ def test_passing_games(game):
 @pytest.mark.parametrize("game", _FAILING_GAMES)
 def test_failing_games(game):
     """Ensures that failing games are still failing."""
-    with pytest.raises(pyspiel.SpielError):
+    with pytest.raises((pyspiel.SpielError, NotImplementedError)):
         test_passing_games(game)
 
 
-@pytest.mark.parametrize("game", _PASSING_GAMES)
-def test_seeding(game):
+def test_seeding():
     """Tests the seeding of the openspiel conversion wrapper."""
     # load envs
-    env1 = pyspiel.load_game(game)
-    env2 = pyspiel.load_game(game)
+    env1 = pyspiel.load_game("2048")
+    env2 = pyspiel.load_game("2048")
```

Comment:
Realistically we want to test all games in case some of them have different rng generators internally, since stuff is written in cpp and can be contributed by external devs anyway.